index.html

```diff
@@ -14,9 +14,9 @@
       <h2 class="mdl-card__title-text">Who are we?</h2>
     </div>
     <div class="mdl-card__supporting-text">
-      Iron Claw is a <a href="http://www.usfirst.org/roboticsprograms/frc">First Robotics Competition</a> (FRC)
-      team run by the Los Gatos High School Robotics Club. Our team number is 972 and our rookie year is 2016!
-      We hope to see all of our fellow teams at Silicon Valley Regional in April!
+      Iron Claw is an FRC team run by the Los Gatos High School Robotics Club.
+      This year (2016), we will restart team 972 at LGHS.
```

Comment:
I'm not sure we need `2016` here. I would either add 2016 as part of the sentence or remove it entirely.